src/proto/grpc/testing/stats.proto

```diff
@@ -41,6 +41,12 @@ message ServerStats {
   // change in server time (in seconds) used by the server process and all
   // threads since last reset
   double time_system = 3;
+
+  // Number of requests received
+  uint64 reqs_received = 4;
+
+  // Number of responses sent
+  uint64 responses_sent = 5;
 }
 
 // Histogram params based on grpc/support/histogram.c
```

Comment:
For HistogramParams

Why aren't we using integers for resolution here? If we're measuring in nanoTime that would work fine and we wouldn't have resolution below 10ns anyway. Measuring in microseconds too would work well too for the levels of performance were concerned with